app/build.gradle

```diff
@@ -34,6 +40,7 @@ android {
 ext {
     ankoVersion = '0.10.6'
     room_version = "2.1.0-alpha04"
+    versionsWork = "1.0.0"
```

Comment:
why do this only for one package? if multiple packages need to be in sync, then only it makes sense to make a variable